### PR TITLE
Digital 5636

### PR DESCRIPTION
**Pull request by stepshep**
Git rid of nodebootstrap, use postgres instead of redis

### CODE REVIEW

**Comment by stepshep** [comment]
Config.js has creds for my local db instance, those will need to be removed before final merge

**Comment by kthayer424** [comment]
go ahead and remove your local credentials.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+// catch 404 and forward to error handler
+app.use(function(req, res, next) {
+  var err = new Error('Not Found');
+  err.status = 404;
+  next(err);
+});
+
+// error handlers
+
+//// development error handler
```
Comment (kthayer424) [inline app.js]: Do these need to be commented out?

**Comment by kthayer424** [comment]
Run jshint against this code and fix those errors

**Comment by stepshep** [comment]
Things looking better yet?

**Pull request closed without merging**